### DISCUSSION TITLE
denoflare -WebAssembly/Wasm -Module-based workers. how can i solve this?

### DISCUSSION THREAD

**Discussion by loglost** [denoland/deno#17688] [Q&A] [answered]
hello, i am just learning dev. i am using denoflare to make my site but got an error with "error: No such file or directory (os error 2).   const valueBytes = await Deno.readFile(localPath);"
my code is Module-based workers sample from https://denoflare.dev/reference/wasm. 
anyone can help me how to solve it plz?

searched :https://stackoverflow.com/questions/72842537/deno-unit-testing-error-no-such-file-or-directory-os-error-2
but seems there is no solution..

**Comment by johnspurlock** [comment, marked as answer]
Hi there - `denoflare` is a separate project from the core `deno` project.  You many want to head over to that project repo and post something there: https://github.com/skymethod/denoflare
